testing/src/test/java/org/motechproject/nms/testing/it/flw/FrontLineWorkerServiceBundleIT.java

```diff
@@ -266,7 +265,7 @@ public void testDeleteRecentInvalidFrontLineWorker() {
         flw.setStatus(FrontLineWorkerStatus.INVALID);
         frontLineWorkerService.update(flw);
 
-        flw = frontLineWorkerService.getByContactNumber(2111111111L);
+        flw = frontLineWorkerService.getById(flw.getId());
         assertEquals(FrontLineWorkerStatus.INVALID, flw.getStatus());
```

Comment:
should we also add an assert here for contact number being null after the update?